blog/content/post/2020-01-10-genode.md

```diff
@@ -0,0 +1,157 @@
+---
+title: 'Genode hypervisor with proprietary RTOS VM - because everybody can
+sometimes have a bad idea'
```

Comment:
@pietrushnic is the title acceptable?

Reply:
@artur-rs This is the final title:
How L4 Genode hypervisor stands against proprietary RTOS solution